modules/core/core-app/src/main/java/com/enonic/xp/core/impl/app/ApplicationServiceImpl.java

```diff
@@ -214,7 +214,7 @@ private Application doInstallLocalApplication( final ByteSource byteSource )
     {
         final ApplicationKey applicationKey = getApplicationKey( byteSource );
 
-        final Application application = installOrUpdateApplication( byteSource, applicationKey, true, false );
+        final Application application = installOrUpdateApplication( byteSource, applicationKey, true, true );
```

Comment:
local apps must not update repository
```suggestion
        final Application application = installOrUpdateApplication( byteSource, applicationKey, true, false );
```